scripts/mk/printvars.mk

```
@@ -6,3 +6,7 @@
 .PHONY: printvars
 printvars: ## Print variable name and values
 	@true $(foreach V, $(sort $(.VARIABLES)),$(if $(filter-out environment% default automatic,$(origin $V)),$(info $V=$(value $V))))
+
+.PHONY: printenvcfg
+printenvcfg: ## Print the environment the resulting exported environment
+	@env | grep -e ^DATABASE_ -e ^KAFKA_ -e ^APP_ -e ^WEB_ -e ^LOGGING_ -e ^CONFIG_PATH= | sort
```

The grep pattern `^APP_` does not match the env var `APP`, because it does not end with an underscore. That's the env var that broke our configuration. Here is a better regular expression with Perl syntax:

```suggestion
	@env | grep -P -e '^(APP|CONFIG_PATH|DATABASE|KAFKA|LOGGING|WEB)(_.*)?=' | sort
```